snippets/serverless-channels-intro.adoc

```diff
@@ -0,0 +1,10 @@
+// Text snippet included in the following files
+//
+// * /serverless/develop/serverless-creating-channels.adoc
+// * /serverless/discover/serverless-channels.adoc
+
+Channels are custom resources that define a single event-forwarding and persistence layer. After events have been sent to a channel from an event source or producer, these events can be sent to multiple Knative services, or other sinks, by using a subscription.
```

Comment:
Do you need the comma after _Knative services_? This reads to me that _other sinks_ are another name for Knative services.